swf/src/avm2/types.rs

```diff
@@ -574,6 +572,3 @@ pub enum Op {
     Timestamp,
     URShift,
 }
-
-#[cfg(target_pointer_width = "64")]
```

Comment:
Undo this, please. Add add something similar to `avm2::op::Op`.

Reply:
I moved the check to `avm2::op::Op` because we now copy all `swf` `Op`s into `avm2` `Op`s, and those are the ones that `Activation` iterates over.